recipes/llama-cpp/all/conanfile.py

```diff
@@ -65,6 +65,9 @@ def export_sources(self):
     def validate(self):
         check_min_cppstd(self, 17 if self._is_new_llama else 11)
 
+        if Version(self.version) >= "b6529" and self.settings.os == "Macos" and Version(self.settings.os.version) < 13:
+            raise ConanInvalidConfiguration(f"{self.ref} requires OSX >=13.")
+
```

Comment:
```suggestion
```

@akos-fulop-public The CI was managed internally to avoid building with Mac, so this error will not occur again in this PR. For users, it should not reach too, because the setting `os.version` is optional and not used by default.